google-cloud-spanner/test/helper.rb

```diff
@@ -149,6 +149,25 @@ def paged_enum_struct response
   end
 end
 
+class MockSpannerHelper
+  def self.expect_execute_streaming_sql mock, results_enum, session_name, sql,
```

Comment:
What do you think about moving this to be an instance method on the existing `MockSpanner` helper class? Then it could be called without a receiver like the other helpers, for example:

```

    mock = Minitest::Mock.new
    expect_execute_streaming_sql mock, results_enum, session.path, sql, transaction: tx_selector, param_types: {}, partition_token: partition_token, options: default_options
    batch_snapshot.session.service.mocked_service = mock
```

Reply:
Since `MockSpanner` has a reference to the `spanner` client object used in the tests, you should even be able to drop the `mock` parameter from your method as follows, as long as you set the mock on the client in the test before calling `expect`. (There are a couple layers of client in this lib, so it might not work in all cases, but you could add references as needed in `MockSpanner`.)

```
  def expect_execute_streaming_sql results_enum, session_name, sql,
                                   transaction: nil, params: nil, param_types: nil, 
                                   resume_token: nil, partition_token: nil, seqno: nil,
                                   options: nil
      spanner.service.mocked_service.expect :execute_streaming_sql, results_enum do |session, sql_query, **kargs|
  ...
```

and 

```

    mock = Minitest::Mock.new
    batch_snapshot.session.service.mocked_service = mock
    expect_execute_streaming_sql results_enum, session.path, sql, transaction: tx_selector, param_types: {}, partition_token: partition_token, options: default_options
  ...
```

Reply:
@quartzmo Thanks for pointing out this. I have updated it.